client/state/initial-state.js

```diff
@@ -16,6 +17,8 @@ import config from 'config';
  */
 const debug = debugModule( 'calypso:state' );
 
+export const MAX_AGE = ms( '7 days' );
```

Comment:
Thanks for the suggestions! I'll update with some constants. The problem with simply adding a comment is that they can get out of sync with the value, which is why I added in the micro lib. I've used it in the past where folks were tired of redefining constants in separate files.